src/constants/translations/ua/become-tutor.json

```diff
@@ -0,0 +1,38 @@
+{
+  "stepLabels": {
+    "generalInfo": "Загальне", 
+    "language": "Мова", 
+    "subjects": "Предмети",
+    "photo": "Фото"
+  },
+  "generalInfo": {
+    "title": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint.",
+    "textFieldLabel": "Коротко опишіть свій професійний статус",
+    "checkboxLabel": "Я підтверджую, що мені більше 18 років",
```

Comment:
As i remember we didn`t use chekbox for age anymore, i forget to remove it, can you?

Reply:
I did it)